ogr/services/gitlab/issue.py

```diff
@@ -96,10 +100,14 @@ def create(
         body: str,
         private: Optional[bool] = None,
         labels: Optional[List[str]] = None,
+        assignees: Optional[List[str]] = None,
     ) -> "Issue":
         data = {"title": title, "description": body}
         if labels:
             data["labels"] = ",".join(labels)
+        if assignees:
+            data["assignee_ids"] = ",".join(list(map(str, assignees)))
```

Comment:
Is the `map` needed? From the type annotation, we should get list of strings or `None`.

Reply:
My bad forgot to convert to string in `def create_issue`